includes/admin/class-wc-stripe-settings-controller.php

```
@@ -234,6 +234,9 @@ public function admin_scripts( $hook_suffix ) {
 			// Show the Stripe Tax banner only if OC is enabled
 			&& $is_oc_enabled;
 
+		// Adaptive Pricing is only available for accounts not based in a European Economic Area country.
+		$is_ap_available_for_account = ! in_array( $this->account->get_account_country(), WC_Stripe_Helper::get_european_economic_area_countries(), true );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Helper' in method 'admin_scripts'. (undefined)

(StaticAccess)